apps/E2E/src/LinkExperimental/specs/ExperimentalLink.spec.win.ts

```diff
@@ -21,6 +21,41 @@ describe('Link Testing Initialization', function () {
   });
 });
 
+describe('Link Testing Functionality', function () {
+  /* Scrolls and waits for the Button to be visible on the Test Page */
+  beforeEach(async () => {
+    await ExperimentalLinkPageObject.scrollToTestElement();
+  });
+
+  it('Validate OnPress() callback was fired -> Click', async () => {
```

Comment:
I know the "->" in the test name is consistent across most tests, but I think we should get rid of it (it's not clear what it means).

Perhaps this would be better:
"Validate OnPress() callback was fired on a Click'"

I can work with Larry to change all instances of it, but starting here would be great :)

Reply:
Yep, sure